axelor-account/src/main/java/com/axelor/apps/account/db/repo/InvoiceManagementRepository.java

```diff
@@ -59,6 +59,7 @@ public Invoice save(Invoice invoice) {
                 .orElse(null);
         invoice.setPaymentDate(latestPaymentDate);
       }
+      invoice.setNextDueDate(Beans.get(InvoiceToolService.class).getNextDueDate(invoice));
```

Comment:
Add this line also in InvoiceToolService.resetInvoiceStatusOnCop (because it's called in InvoiceManagementRepository.copy)